File: campus-quest/logo/logo.py
from serply import Serply

serply = Serply('N4shxJsBsGLLvTM9VQvKxXdn')

def getImage(college):
    results = serply.image(college + "logo wikimedia transparent background")
    imgs = results["image_results"]
    for i in imgs:
        if "commons.wikimedia.org" in i["link"]["domain"]:
            print("WIKIMEDIA FOUND")
            return i["link"]["href"]
            
    
getImage("Cornell University")

print('Hello from python')

File: campus-quest/logo.js
// Import the Serply module
const Serply = require('serply');

// Initialize Serply with your API key
const serply = new Serply('N4shxJsBsGLLvTM9VQvKxXdn');

// Function to get the logo image link from Wikimedia for a given college
async function getImage(college) {
  // Get image results from Serply
  const results = await serply.image(college + "logo wikimedia transparent background");
  const imgs = results.image_results;

  // Iterate through the image results
  for (const img of imgs) {
    if (img.link.domain.includes("commons.wikimedia.org")) {
    //   console.log("WIKIMEDIA FOUND");
      console.log(img.link.href)
      return img.link.href;
    }
  }
}

// Call the function with the college name
getImage("Cornell University")
  .then((logoLink) => console.log(logoLink))
  .catch((error) => console.error(error));
